File: Arrays-and-Hashing/Closest to 0/C++/main.cpp
class Solution {
public:
    int findClosestNumber(std::vector<int>& nums) 
    {
        int closest = nums[0];

        for(int num : nums)
        {
            if(abs(num) < abs(closest))
                closest = num;
            if(abs(num) == abs(closest))
                closest = std::max(closest, num);
        }
        return closest;
    }
};

File: Arrays-and-Hashing/Closest to 0/Python/main.py
class Solution:
    def findClosestNumber(self, nums: List[int]) -> int:
        closest = nums[0]
        
        for num in nums:
            if(abs(num) < abs(closest)):
                closest = num
            if(abs(num) == abs(closest)):
                closest = max(closest, num)
        
        return closest